Jenkinsfile.yaml

```diff
@@ -16,14 +16,16 @@ agent: any
 
 stages:
     - name: First
-      scripts:
-        -   ./scripts/hello
+      steps:
+        - sh './scripts/hello'
```

Comment:
A few things:
It looks like there are 2 formats here.  The condensed step command and parameter in one line, and the split format on line 22 & 27.  They should be consistent.

I know you probably made this match the format for yaml parser steps, but I think the previous form of defining a script makes more sense from a user's perspective.  It's just going to be some extra code to handle this case.  This is where defining the final yaml form is going to be incredibly important.  For now: I vote to get the flow working.  Then, as it gets used in this coding phase, to me it makes sense to tweak the yaml definition before it gets widespread use.

Reply:
In the step format in line 22. I am unable to get actual type of the default parameter. I want to go with the format in line 22, but at this point in time, it is working for only string type.

Reply:
Not urgent, but IMO if we replicate the declarative pipeline syntax as yaml, we have not accomplished the simplification aimed by this plugin. This is something to think about for the formal yaml schema.

Reply:
Exactly, I liked how it was before the change; where basically it pointed out the commands to run while under the covers the conversion was to the declarative syntax.  IMO, this newer format fits the goal of "pipeline syntax in yaml".  For simplification, the yaml needs to be something incredibly short with the plugin doing as much of the processing work as possible.